disnake/interactions/base.py

```diff
@@ -178,7 +178,6 @@ def __init__(self, *, data: InteractionPayload, state: ConnectionState):
         self.guild_locale: Optional[str] = data.get("guild_locale")
         # one of user and member will always exist
         self.author: Union[User, Member] = MISSING
-        self._permissions: int = 0
```

Comment:
`self._permissions` is possibly unbound

Reply:
https://github.com/DisnakeDev/disnake/pull/321/commits/40f10fb5c323876ebc9c85459b4a110b80ce8ade